eng/jacoco-test-coverage/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-ai-metricsadvisor</artifactId>
-      <version>1.1.8</version> <!-- {x-version-update;com.azure:azure-ai-metricsadvisor;current} -->
+      <version>1.1.9</version> <!-- {x-version-update;com.azure:azure-ai-metricsadvisor;current} -->
```

Comment:
Increment versions PRs on `main` bump libraries to a beta for the next minor version.
```suggestion
      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-metricsadvisor;current} -->
```